chat.js

```diff
@@ -350,6 +353,38 @@ class CommandContext {
 		}
 		return true;
 	}
+	checkR9k(user, room, message) {
+		let checksum = Chat.md5(toId(message)).substr(0, 16);
```

Comment:
Let's use the entire digest; I don't think the RAM use will be appreciably higher and it'll be more futureproof.